kubernetes/apps/external-secrets.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersion: argoproj.io/v1alpha1
+kind: ApplicationSet
+metadata:
+  name: external-secrets
+spec:
+  goTemplate: true
+  generators:
+    # targets all clusters
```

Comment:
How are clusters defined? I see that we have the `gke-prow` cluster defined, but I don't see the `gke-utility` cluster defined and labeled. Is it done manually/via click-ops?

Reply:
gke-utility is in the default cluster. it was missing labels so I renamed it in the Argo Console and added the labels you see on gke-prow

Reply:
Okay, thanks for the clarification! I think we should document this as part of the README.